src/main/java/core/basesyntax/BallCollor.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public enum BallCollor {
```

Comment:
The enum name 'BallCollor' seems to be a typo. According to Java naming conventions, class and enum names should be written in CamelCase and should be descriptive and correctly spelled. Consider renaming it to 'BallColor'.